dao/src/main/java/com/iluwatar/dao/CustomerDao.java

```diff
@@ -9,9 +9,9 @@
  */
 public interface CustomerDao {
 	
-    public List<Customer> getAllCustomers();
-    public Customer getCusterById(int id);
-    public void addCustomer(Customer customer);
-    public void updateCustomer(Customer customer);
-    public void deleteCustomer(Customer customer);
+    List<Customer> getAllCustomers();
+    Customer getCustomerById(final int id);
```

Comment:
Suggestion: I don't think we need to document parameters to be `final`. It increases verbosity.